packages/web3-eth-miner/types/index.d.ts

```diff
@@ -17,13 +17,12 @@
  * @date 2019
  */
 
-import {Accounts} from 'web3-eth-accounts';
 import {provider} from 'web3-providers';
-import {AbstractWeb3Module, Web3ModuleOptions, Hex} from 'web3-core';
+import {AbstractWeb3Module, Web3ModuleOptions} from 'web3-core';
 import * as net from 'net';
```

Comment:
Here we are making net as a dependency of this module. Only for typing. I think we can remove this. What do you say @joshstevens19, @nivida?

Reply:
It's the node net module and not the web3 net module that's why I think this would be okay.